packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -306,7 +305,7 @@ const participantStateStringTrampoline = (
 ): string | undefined => {
   /* @conditional-compile-remove(one-to-n-calling) */
   /* @conditional-compile-remove(PSTN-calls) */
-  return props.participantState === 'Idle' || props.participantState === 'Connecting'
+  return props.participantState === 'Connecting'
```

Comment:
I still feel we should remove this. Connecting shouldn't mean "Calling..." as not all participants who are connecting are being called.
This is a pure UI component so it shouldn't need to adhere to calling's decision of what a connecting participant means, I think instead in the selector we should be mapping a connecting + phoneNumber participant to be in the ringing state
Let me know your thoughts on this

Reply:
I like your suggestion to map a connecting PSTN participant to 'ringing' and removing this. before we were operating on the knowledge that PSTN participants never hit the ringing state in the SDK because of network limitations.